classes/render/assets.php

```diff
@@ -1049,14 +1049,14 @@ protected static function field_script_dependencies()
 	 *
 	 * @return string
 	 */
-	protected static function get_validator_locale_url($locale)
+	public static function get_validator_locale_url($locale)
 	{
-		if(!empty($locale)){
-			$locale = self::set_locale_code($locale);
-		}
+		//Use correct formatted locale for file to be matched
+		$locale = self::set_locale_code($locale);
```

Comment:
Why are we removing a check here?

Reply:
@mrpritchett The check was removed to always run set_locale_url() which sets a default $locale to "en" if no locale was set or no file matching local found